src/Stripe.net/Services/Subscriptions/SubscriptionItemOptions.cs

```diff
@@ -5,6 +5,14 @@ namespace Stripe
 
     public class SubscriptionItemOptions : IHasId, INestedOptions, IHasMetadata
     {
+        /// <summary>
+        /// Define thresholds at which an invoice will be sent, and the subscription advanced to a
+        /// new billing period. When updating, pass an empty string to remove previously-defined
+        /// thresholds.
+        /// </summary>
+        [JsonProperty("billing_thresholds")]
+        public SubscriptionItemBillingThresholdsOptions BillingThresholds { get; set; }
```

Comment:
technically you're introducing something you know you will break later with "no sharing" right?

Reply:
yeah this one is really odd because the unshared and shared name is the same